index.js

```diff
@@ -380,7 +380,8 @@ export const Classes = {
 		quizAttemptList: 'quiz-attempt-list',
 		recommended: 'recommended',
 		startDate: 'start-date',
-		timeLimitEnforced: 'time-limit-enforced'
+		timeLimitEnforced: 'time-limit-enforced',
+		notimelimit: 'notimelimit'
```

Comment:
Can this be standardized with the others? `noTimeLimit: 'no-time-limit'`?

Reply:
@marcsoiferman How important is it for this to be standardized? This constant is used in other PRs that are already merged.

Reply:
In that case, not important